### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by CactiStaccingCrane**
Address issue raised in #2008, made a new PR because I couldn't run pre-commit for some reason

### CODE REVIEW

File: .gitignore
Comment (olliestanley) [inline .gitignore:18]: I think instead of all these you can just have one line: `.obsidian/`

**Review by olliestanley — COMMENTED** [review]
Thanks, this should help new developers understand the repo better :) Just left one comment on how the gitignore could be simplified a little

File: .gitignore
Comment (CactiStaccingCrane) [reply .gitignore:18]: done

**Review by olliestanley — APPROVED** [review]
This will still need a code owner review from Yannic or Andreas as it touches a root file (`.gitignore`) but the docs changes look good, thanks!

**Review by andrewm4894 — APPROVED** [review]

**Review by andrewm4894 — APPROVED** [review]

**Review by yk — COMMENTED** [review]
thank you!

**Review by yk — APPROVED** [review]

**Pull request merged**
